cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java

```diff
@@ -419,6 +419,8 @@ protected void handleGetSubscriptions(HttpServletRequest request, HttpServletRes
         // get the parameters to be updated
         String ngsiVersion = request.getParameter("ngsi_version");
         String subscriptionID = request.getParameter("subscription_id");
+        String fiwareService = request.getHeader("Fiware-service");
```

Comment:
Despite it is not a problem for the html headers parser, FIware-Service is better than Fiware-service